resources/views/layouts/parts/navigation/sidebar/301-overview.blade.php

```diff
@@ -0,0 +1,10 @@
+@can('admin.overview.read')
```

Comment:
please rename file to 350 or something, if you cal it 301, no one can get between 300 and 301